### ISSUE TITLE
Expose current location cache

### ISSUE THREAD

**Issue by ChrisBr** [InfluxCommunity/influxdb-rails#59]
Shall we expose and document the location cache implemented in: https://github.com/influxdata/influxdb-rails/pull/53/commits/4226a5ceda34f409e34494ee20f848031a01ffad

I think this could be potentially useful for people implementing their own subscriber without the need to dig into the code. Thinking about a method / methods like:

```
InfluxDB::Rails.current_location # -> UsersController#index
InfluxDB::Rails.current_action # -> action
InfluxDB::Rails.current_controller # -> UsersController
```

**Comment by dmke** [comment]
That's a good idea.

**Comment by ChrisBr** [comment]
Ok, I will provide a PR for that then.

**Comment by dmke** [comment]
Maybe create a class with this interface:

```ruby
class InfluxDB::Rails::Context
  def location; end
  def action;  end
  def controller; end
  def reset; end
end
```

and expose it via

```ruby
def InfluxDB::Rails.current
  @current ||= Context.new
end
```

We could then (later) easily define a `current.extra = hash` and make it available for users to solve #36. Similar to how `sentry-raven` works: https://github.com/getsentry/raven-ruby/blob/master/lib/raven/context.rb.
